ui/src/components/credentials/IssuedTable.tsx

```diff
@@ -1,53 +1,302 @@
+    [setSearchParams]
+  );
+
+  const handleStatusChange = ({ target: { value } }: RadioChangeEvent) => {
+    const parsedCredentialStatus = credentialStatusParser.safeParse(value);
+    if (parsedCredentialStatus.success) {
+      const params = new URLSearchParams(searchParams);
+
+      if (parsedCredentialStatus.data === "all") {
+        params.delete(STATUS_SEARCH_PARAM);
+      } else {
+        params.set(STATUS_SEARCH_PARAM, parsedCredentialStatus.data);
+      }
+
+      setSearchParams(params);
+    } else {
+      processZodError(parsedCredentialStatus.error).forEach((error) => void message.error(error));
+    }
+  };
+
+  useEffect(() => {
+    const { aborter } = makeRequestAbortable(fetchCredentials);
+
+    return aborter;
+  }, [fetchCredentials]);
+
   return (
     <TableCard
       defaultContents={
         <>
           <Avatar className="avatar-color-cyan" icon={<IconCreditCardRefresh />} size={48} />
 
           <Typography.Text strong>No credentials</Typography.Text>
+
           <Typography.Text type="secondary">
             Issued credentials will be listed here.
           </Typography.Text>
 
-          <Link to={generatePath(ROUTES.issueCredential.path)}>
-            <Button icon={<IconCreditCardPlus />} type="primary">
-              {ISSUE_CREDENTIAL}
-            </Button>
-          </Link>
+          {credentialStatus === "all" && (
+            <Link to={generatePath(ROUTES.issueCredential.path)}>
+              <Button icon={<IconCreditCardPlus />} type="primary">
+                {ISSUE_CREDENTIAL}
+              </Button>
+            </Link>
+          )}
         </>
       }
-      isLoading={false}
-      onSearch={() => null}
-      query={null}
+      isLoading={isAsyncTaskStarting(credentials)}
+      onSearch={onSearch}
+      query={queryParam}
       searchPlaceholder="Search credentials, attributes, identifiers..."
-      showDefaultContents={true}
+      showDefaultContents={showDefaultContent}
       table={
         <Table
-          dataSource={[]}
+          columns={tableColumns.map(({ title, ...column }) => ({
+            title: (
+              <Typography.Text type="secondary">
+                <>{title}</>
+              </Typography.Text>
+            ),
+            ...column,
+          }))}
+          dataSource={credentialsList}
+          locale={{
+            emptyText:
+              credentials.status === "failed" ? (
+                <ErrorResult error={credentials.error.message} />
+              ) : (
+                <NoResults searchQuery={queryParam} />
+              ),
+          }}
           pagination={false}
           rowKey="id"
           showSorterTooltip
           sortDirections={["ascend", "descend"]}
         />
       }
       title={
-        <Row justify="space-between">
-          <Space size="middle">
-            <Card.Meta title={ISSUED} />
+        <Row align="middle" justify="space-between">
+          <Space align="end" size="middle">
```

Comment:
Why are `align` needed in Row and Space? It results in a non-centered meta title. Also, it's not present in `LinksTable`.

[Screencast from 12-04-23 16:50:55.webm](https://user-images.githubusercontent.com/2691584/231496647-263b571b-7edb-4e6e-9bdd-601a5144ddac.webm)

Reply:
I'm going to add them to linkTable because is looking better
![align](https://user-images.githubusercontent.com/19190248/231520401-f1d6436b-35b1-4230-afb9-653d9e0a53c7.gif)

Reply:
After a call, we agreed to avoid props and override the `.ant-tag` with `padding-top: 4px;` and/or investigate theming options to better accommodate natural framework alignments.